internal/backend/azure/azure.go

```diff
@@ -245,12 +261,17 @@ func (be *Backend) saveSmall(ctx context.Context, objName string, rd backend.Rew
 		return errors.Wrap(err, "StageBlock")
 	}
 
+	commitBlockListOptions := &blockblob.CommitBlockListOptions{}
+	if accessTier != nil {
```

Comment:
the `if` is redundant. If accessTier is nil, then setting it is a noop.

Reply:
Done. I have removed the redundant if.